src/Redax/Crm-add-new-product/slices/product-slice.jsx

```diff
@@ -0,0 +1,92 @@
+import { createSlice } from "@reduxjs/toolkit";
+import { createNewProduct, addImages, addPrice } from "../operation/Operation";
+
+export const productSlice = createSlice({
+  name: "newProduct",
+  initialState: {
+    productId: null,
+    name: null,
+    description: null,
+    product_status: "new",
+    main_category: null,
+    sub_categories: [],
+    price: [],
+    isLoading: null,
+    operation: null,
+    error: null,
+    formErrors: {},
+    imagesUploadCount: 0,
+    images: false
```

Comment:
it will be better to separate this state as a `const PRODUCT_INITIAL_STATE = {...}`

Reply:
done